src/vcpkg/commands.build.cpp

```diff
@@ -1019,6 +1019,10 @@ namespace vcpkg
         {
             return m_paths.scripts / "toolchains/windows.cmake";
         }
+        else if (cmake_system_name == "visionOS")
+        {
+            return m_paths.scripts / "toolchains/visionos.cmake";
```

Comment:
There is no such thing: https://github.com/microsoft/vcpkg/tree/master/scripts/toolchains

I think you need a separate PR adding that over there?

Reply:
Yes, sorry. This PR got opened first so the one over there references this one. Wasn't sure how to stage it. :)